docs/next/modules/en/pages/troubleshooting/troubleshooting.adoc

```diff
@@ -0,0 +1,300 @@
+serviceaccount/default                                                                    capi-clusters  23h  
+rke2config.bootstrap.cluster.x-k8s.io/cluster1-mp-0-dm59p                                capi-clusters  32s  
+rke2config.bootstrap.cluster.x-k8s.io/cluster1-mp-1-gv6kh                                capi-clusters  32s  
+rke2configtemplate.bootstrap.cluster.x-k8s.io/cluster1-pool0                             capi-clusters  33s  
+rke2configtemplate.bootstrap.cluster.x-k8s.io/cluster1-pool1                             capi-clusters  33s  
+clusterclass.cluster.x-k8s.io/clusterclass1                                               capi-clusters  33s  
+cluster.cluster.x-k8s.io/cluster1                                                        capi-clusters  32s  
+machinepool.cluster.x-k8s.io/cluster1-mp-0-d4fdv                                         capi-clusters  32s  
+machinepool.cluster.x-k8s.io/cluster1-mp-1-l86kv                                         capi-clusters  32s  
+clustergroup.fleet.cattle.io/clusterclass1                                                capi-clusters  33s  
+azureclusteridentity.infrastructure.cluster.x-k8s.io/cluster-identity                     capi-clusters  33s  
+azuremanagedcluster.infrastructure.cluster.x-k8s.io/cluster1-l2cs6                       capi-clusters  32s  
+azuremanagedclustertemplate.infrastructure.cluster.x-k8s.io/cluster                     capi-clusters  33s  
+azuremanagedcontrolplane.infrastructure.cluster.x-k8s.io/cluster1-rv8v4                  capi-clusters  32s  
+azuremanagedcontrolplanetemplate.infrastructure.cluster.x-k8s.io/cluster1-control-plane  capi-clusters  33s  
+azuremanagedmachinepool.infrastructure.cluster.x-k8s.io/cluster1-mp-0-78tck              capi-clusters  32s  
+azuremanagedmachinepool.infrastructure.cluster.x-k8s.io/cluster1-mp-1-zdscw              capi-clusters  32s  
+azuremanagedmachinepooltemplate.infrastructure.cluster.x-k8s.io/cluster1-pool0           capi-clusters  33s  
+azuremanagedmachinepooltemplate.infrastructure.cluster.x-k8s.io/cluster1-pool1           capi-clusters  33s  
+----
++
+. Then you can check what the relation is between all object resources
++
+[source,bash]
+----
+kubectl lineage -n capi-clusters cluster.cluster.x-k8s.io/cluster1
+----
++
+Output:
++
+image::lineage-cluster-output.png[Output]
+
+== How to enable debug mode for Rancher Turtles and CAPI operators
+
+The helm chart exposes values for increasing the log level via the usual `values.yaml` configuration parameters:
+
+* Cluster API Operator - log level for CAPI operator can also be increased using helm chart, if installed with Rancher Turtles chart use:
+
+[source,bash]
+----
+        --set cluster-api-operator.logLevel=5
+        --set rancherTurtles.managerArguments[0]="-v=5"
+----
+
+For example:
+
+[source,bash]
+----
+helm upgrade rancher-turtles turtles/rancher-turtles \
+            -n rancher-turtles-system \
```

Comment:
```suggestion
            -n rancher-turtles-system \
            --reuse-values \
```

To keep users safe from overriding selected custom settings.

Reply:
great idea